libkbfs/root_metadata.go

```diff
@@ -280,6 +331,242 @@ func (md *RootMetadata) swapCachedBlockChanges() {
 	}
 }
 
+// GetTLFCryptKeyParams wraps the respective method of the underlying BareRootMetadata for convenience.
```

Comment:
I didn't recall and the only design note I had about it was a comment from you saying it was ideal but that was before making it an interface. So I unembedded it but then ran into an issue with `RootMetadata.deepCopy`. It would have made that really awkward so I decided to just leave it unembedded after all.